CONTRIBUTING.md

```diff
@@ -1,24 +1,3 @@
 # Contributing
 
-🙏 Thanks for your interest in contributing!
-
-## Setup
-
-After cloning the repo, run the following command to install dependencies:
-
-```sh
-yarn
-```
-
-This project uses [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to manage multiple packages within one repository. Check out the `workspaces` key within `package.json` to view all possible workspace directories. Some workspaces of note:
-
-- `packages/gatsby-mdx`: the source for the gatsby-mdx plugin
-- `examples/docs`: the gatsby-mdx docs site (built with [Gatsby](https://next.gatsbyjs.org/))
-
-## Submitting a bug report
-
-To submit a bug report, follow [this link](https://github.com/ChristopherBiscardi/gatsby-mdx/issues/new?template=bug_report.md), which will prepopulate a new GitHub issue with the bug report template.
-
-## Submitting a feature request
-
-To submit a feature request, follow [this link](https://github.com/ChristopherBiscardi/gatsby-mdx/issues/new?template=feature_request.md), which will prepopulate a new GitHub issue with the feature request template.
+See [docs/contributing.mdx](./examples/docs/content/contributing.mdx).
```

Comment:
Figured it would be good to keep the root level CONTRIBUTING.md, as GitHub will show a prompt in the UI that links here if you are attempting to submit your first issue or PR. 👍